Package.swift

```diff
@@ -17,37 +13,26 @@ let package = Package(
         // watchOS is unsupported due to lack of CoreImage support
     ],
     products: [
+        .library(name: "BarcodeHero", targets: ["BarcodeHeroCore", "BarcodeHeroUI"]),
         .library(name: "BarcodeHeroCore", targets: ["BarcodeHeroCore"]),
-        // BLOCKED BY SR-2866 -- Uncomment
-        // .library(name: "BarcodeHeroUI", targets: ["BarcodeHeroUI"]),
-        .library(name: "BarcodeHero", targets: ["BarcodeHeroCore"]),
-        // BLOCKED BY SR-2866 -- Uncomment and delete the previous definition of the BarcodeHero library product
-        // .library(name: "BarcodeHero", targets: ["BarcodeHeroCore", "BarcodeHeroUI"]),
+        .library(name: "BarcodeHeroUI", targets: ["BarcodeHeroUI"]),
     ],
     dependencies: [],
     targets: [
         .target(
             name: "BarcodeHeroCore",
             dependencies: []
         ),
-        // BLOCKED BY SR-2866 -- Uncomment
-        // .target(
-        //     name: "BarcodeHeroUI",
-        //     dependencies: []
-        // ),
+        .target(
+            name: "BarcodeHeroUI",
+            dependencies: []
+        ),
         .testTarget(
             name: "BarcodeHeroCoreTests",
             dependencies: [
                 .target(name: "BarcodeHeroCore"),
             ]
         ),
-        // BLOCKED BY SR-2866 -- Uncomment
```

Comment:
worth leaving this if we havent implemented the test target yet?

Reply:
There probably won't be a `BarcodeHeroUITests` target though, so this was my bad. It's not dependent on Resources anymore either.